aspnetcore/tutorials/dotnet-watch.md

```diff
@@ -1,195 +1,98 @@
 ---
-title: Developing ASP.NET Core applications using dotnet watch | Microsoft Docs
+title: Developing ASP.NET Core apps using dotnet watch | Microsoft Docs
 author: rick-anderson
-description: 
-keywords: ASP.NET Core,
+description: Shows how to use dotnet watch.
+keywords: ASP.NET Core, using dotnet watch
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 02/14/2017
 ms.topic: article
 ms.assetid: 563ffb3f-d369-4aa5-bf0a-7300b4e7832c
 ms.technology: aspnet
 ms.prod: aspnet-core
 uid: tutorials/dotnet-watch
 ---
-# Developing ASP.NET Core applications using dotnet watch
+# Developing ASP.NET Core apps using dotnet watch
 
 <a name=dotnet-watch></a>
 
-By [Victor Hurdugaci](https://twitter.com/victorhurdugaci)
+By [Victor Hurdugaci](https://twitter.com/victorhurdugaci) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-## Introduction
+`dotnet watch` is a tool that runs a `dotnet` command when source files change.  For example, when a file changes it can compile your code, run tests, or publish your app.
 
-`dotnet watch` is a development time tool that runs a `dotnet` command when source files change. It can be used to compile, run tests, or publish when code changes.
+In this tutorial we'll use an existing Web API app with two endpoints; one that returns a sum and one that returns a product. The product method contains an intentional bug that we'll fix as part of this tutorial.
 
-In this tutorial we'll use an existing WebApi application that calculates the sum and product of two numbers to demonstrate the use cases of `dotnet watch`. The sample application contains an intentional bug that we'll fix as part of this tutorial.
+Download the [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/tutorials/dotnet-watch/sample). It contains two projects, `WebApp` (a web app) and `WebAppTests` (unit tests for the web app).
```

Comment:
Given that this tutorial depends on the reader doing a download, might be worth adding instructions (i.e., download the entire repo and navigate to the right folder)